README.md

```diff
@@ -1,2 +1,37 @@
-# odh-manifests
-A repository for Open Data Hub Kustomize manifests extending upstream Kubeflow manifests
+# Open Data Hub Manifests
+A repository for [Open Data Hub](https://opendatahub.io) Kustomize manifests extending upstream Kubeflow manifests.
+
+## Community
+
+* Website: https://opendatahub.io
+* Documentation: https://opendatahub.io/docs.html (applicable for version 0.5.1)
+* Mailing lists: https://opendatahub.io/community.html
+* Community meetings: https://gitlab.com/opendatahub/opendatahub-community
+
+## Components
+
+Open Data Hub provides various AI/ML and Data Science tools on top of OpenShift Container Platform. The components currently available are:
+
+* [JupyterHub](jupyterhub/README.md)
+* Spark Operator
+* Kafka
+* Superset
+* AI Library
+* Grafana & Prometheus
+
+Some components are still in process of conversion from Ansible Operator based version on [Gitlab](https://gitlab.com/opendatahub/opendatahub-operator/)
+
+* Argo
+* Seldon
+* Data Catalog
+    * Hue
+    * Hive
+    * Thrift Server
+
+There are also other components planned to be added
+
+* Airflow
+
+## Deploy
+
+Since we are relying on [Kustomize](https://github.com/kubernetes-sigs/kustomize), [kfctl](https://github.com/kubeflow/kfctl) and [Kubeflow Operator](https://github.com/kubeflow/kfctl/blob/master/operator.md) for deployment, you can follow the documentation at [Kubeflow.org](https://www.kubeflow.org/docs/openshift/). The only change is to use this repository instead of Kubeflow manifests.
```

Comment:
I think we need to re-word this too. "There are two options to install
1. Using kfctl tool ....
2. Using Operator ...."